apps/fabric/metro.config.js

```diff
@@ -22,6 +24,7 @@ const config = {
   // We need to make sure that only one version is loaded for peerDependencies
   // So we exclude them at the root, and alias them to the versions in example's node_modules
   resolver: {
+    assetExts: [...defaultConfig.resolver.assetExts, 'lottie'],
```

Comment:
Do you think it would be helpful to mark new parts that need to be documented like this with TODO? ex:
```suggestion
    //TODO: Document For dotLottie
    assetExts: [...defaultConfig.resolver.assetExts, 'lottie'],
```

Reply:
Yeah but that's the only part I guess.

Reply:
Updated the docs as well.